sw/device/silicon_owner/tock/tests/uart/BUILD

```diff
@@ -30,19 +30,17 @@ tock_elf2tab(
 
 tock_image(
     name = "image",
-    app_flash_start = 0x20040000,
     apps = [":tab"],
+    exec_env = "//hw/top_earlgrey:silicon_owner_sival_rom_ext",
     kernel = "//sw/device/silicon_owner/tock/kernel",
+    manifest = "//sw/device/silicon_owner:manifest",
```

Comment:
FYI, the `exec_env` usually supplies the manifest (ie: the binary and test rules typically leave it blank and fallback to the `exec_env`'s manifest unless they're trying to do something special for a test).

Reply:
`manifest` is now optional and I've removed its specifications.